src/linear_algebra/affine_space/independent.lean

```diff
@@ -245,6 +268,28 @@ begin
   simp [hf]
 end
 
+/-- If a set of points is affinely independent, so is any subset. -/
+lemma affine_independent_subset_of_affine_independent {s t : set P}
+  (ha : affine_independent k (λ x, x : t → P)) (hs : s ⊆ t) :
+  affine_independent k (λ x, x : s → P) :=
+begin
+  let f : s → t := λ x, ⟨x, hs x.property⟩,
+  let fe : s ↪ t := ⟨f, λ x y h, begin
+    rw subtype.ext_iff,
+    rw subtype.mk_eq_mk at h,
+    exact h
+  end⟩,
```

Comment:
I think this is called `set.embedding_of_subset`.

Reply:
Thanks, now using that.